pkg/analysis_server/lib/src/services/correction/fix.dart

```diff
@@ -194,6 +195,8 @@ class DartFixKind {
       const FixKind('MAKE_CLASS_ABSTRACT', 50, "Make class '{0}' abstract");
   static const REMOVE_DEAD_CODE =
       const FixKind('REMOVE_DEAD_CODE', 50, "Remove dead code");
+  static const REPLACE_WITH_COLLECTION_LITERAL = const FixKind(
+      'REPLACE_WITH_COLLECTION_LITERAL', 50, 'Replace with collection literal');
```

Comment:
nit: It occurs to me that we could drop the word "collection", at least from the description, because it really doesn't add anything (there isn't any other kind of literal that could replace an instance creation of any of these classes).